src/debugger/debugAdapterFactory.ts

```diff
@@ -152,7 +152,8 @@ export class LLDBDebugConfigurationProvider implements vscode.DebugConfiguration
         if (
             this.platform === "win32" &&
             launchConfig.testType === undefined &&
-            path.extname(launchConfig.program) !== ".exe"
```

Comment:
Is there not still value in this check? Or does something else prevent us from getting `.exe.exe`?